dist/index.html

```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Leaderboard App</title>
+<script defer src="runtime.bundle.js"></script><script defer src="index.bundle.js"></script></head>
+<body>
+    <section id="leaderboard">
+        <h1>Leaderboard</h1>
+
+        <!-- Recent Score Section -->
+        <div id="recent-score" class="row">
+            <div class="column">
+                <h2>Recent Scores</h2>
+                <button id="refresh-button">Refresh</button>
+                <ul id="recent-score-list"></ul>
+            </div>
+
+            <!-- Name and Score Input -->
+            <div class="column">
+                <h2>Add Your Score</h2>
+                <form id="score-form">
+                    <input type="text" id="player-name" placeholder="Your Name" required>
+                    <input type="number" id="player-score" placeholder="Your Score" required>
+                    <button type="submit">Submit</button>
+                </form>
+            </div>
+        </div>
+    </section>
```

Comment:
| @DevKaranJ Design | Microverse Design |
|-|-|
| ![image](https://github.com/DevKaranJ/Leaderboard/assets/93039359/2c8f96df-a1b1-46e4-9721-899b18200db4) | ![image](https://github.com/DevKaranJ/Leaderboard/assets/93039359/a8141ff4-0339-4f9b-9fd9-2a49cfd4bec9) |

- [ ] It would be nice if you make your design look similar to the given microverse wireframe to follow the project requirements 😊 Kindly focus on the below points ⬇️

1. Kindly make the **Recent Scores** heading and **Refresh** button in one line 👍
2. Kindly don't expand submit the button and make it expand as you can see in the given wireframe 👍